dockstore-webservice/src/main/java/io/dockstore/webservice/core/Partner.java

```diff
@@ -0,0 +1,14 @@
+package io.dockstore.webservice.core;
+
+/**
+ * The frontend will need to convert the enums to friendly values
+ */
+public enum Partner {
+    GALAXY,
+    TERRA,
+    DNA_STACK,
+    DNA_NEXUS,
+    CGC,
+    NHLBI_BIODATA_CATALYST,
+    ANVIL
```

Comment:
Need Cavatica and Nextflow Tower? I haven't seen how you use this yet.

Reply:
Tower for sure